proxysql-scheduler/jenkins/proxysql-scheduler-golang.yml

```diff
@@ -0,0 +1,15 @@
+- job:
+    name: proxysql-scheduler-go-pxc80-pipeline
+    project-type: pipeline
+    description: |
+        Do not edit this job through the web!
+    concurrent: true
```

Comment:
After `concurrent: true`, please specify all parameters you've defined in groovy file here as well, example:
```
    parameters:
    - string:
        name: GIT_REPO
        default: "https://github.com/percona/percona-server"
        description: URL to percona-server repository
    - string:
        name: BRANCH
        default: "5.7"
        description: Tag/Branch for percona-server repository
```

Reply:
done